main.tf

```diff
@@ -19,19 +19,36 @@ locals {
 # VPC
 ###################
 
+data "aws_availability_zones" "available" {}
+
+data "aws_vpc" "cq_vpc" {
+  id = local.vpc_id
+  depends_on = [
```

Comment:
I am not sure that this `depends_on` is necessary because Terraform will understand the ordering anyway.

Reply:
removed